controllers/ocsinitialization/sccs.go

```diff
@@ -103,44 +104,13 @@ func newRookCephSCC(namespace string) *secv1.SecurityContextConstraints {
 }
 
 func newRookCephCSISCC(namespace string) *secv1.SecurityContextConstraints {
-	scc := blankSCC()
-
-	scc.Name = "rook-ceph-csi"
-	scc.AllowPrivilegedContainer = true
-	scc.AllowHostNetwork = true
-	scc.AllowHostDirVolumePlugin = true
-	scc.AllowedCapabilities = []corev1.Capability{
-		secv1.AllowAllCapabilities,
-	}
-	scc.AllowHostPorts = true
-	scc.AllowHostPID = true
-	scc.AllowHostIPC = true
-	scc.ReadOnlyRootFilesystem = false
-	scc.RequiredDropCapabilities = []corev1.Capability{}
-	scc.DefaultAddCapabilities = []corev1.Capability{}
-	scc.RunAsUser = secv1.RunAsUserStrategyOptions{
-		Type: secv1.RunAsUserStrategyRunAsAny,
-	}
-	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
-		Type: secv1.SELinuxStrategyRunAsAny,
-	}
-	scc.FSGroup = secv1.FSGroupStrategyOptions{
-		Type: secv1.FSGroupStrategyRunAsAny,
-	}
-	scc.SupplementalGroups = secv1.SupplementalGroupsStrategyOptions{
-		Type: secv1.SupplementalGroupsStrategyRunAsAny,
-	}
-	scc.Volumes = []secv1.FSType{
-		secv1.FSTypeAll,
-	}
-	scc.Users = []string{
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-plugin-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-provisioner-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-rbd-attacher-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-cephfs-plugin-sa", namespace),
-		fmt.Sprintf("system:serviceaccount:%s:rook-csi-cephfs-provisioner-sa", namespace),
+	rookValues := cephcsi.SecurityContextConstraintsValues{
+		Namespace: "rook-ceph",
+		Deployer:  "rook",
```

Comment:
this should be `rook-` because the name is `rook-ceph-csi` and all the SCC Users are starts with `rook-`

Reply:
This needs to be without the `-`, it will be added automatically when `Deployer` is set:

https://github.com/ceph/ceph-csi/blob/574852e27c84b8ac78ad7412b0f8ca300eb2bb4b/api/deploy/ocp/scc.go#L93-L95